### ISSUE TITLE
DI Activation

### ISSUE THREAD

**Issue by grandchamp** [grandchamp/Identity.Dapper#4]
On **Identity.Dapper.SqlServer\Extensions\ServiceCollectionExtensions.cs** there is an commented implementation with generics in **AddStores** that i can't make work.
I always get DI activation exception.

**Comment by grandchamp** [comment]
And currently, the **AddDapperIdentityForSqlServer** only works with

```
services.AddIdentity<DapperIdentityUser, DapperIdentityRole<int>>
```

**Comment by grandchamp** [comment]
Fixed in #5
